lib/classes/ConfigSchemaHandler/normalizeAjvErrors.test.js

```diff
@@ -0,0 +1,563 @@
+'use strict';
+const Ajv = require('ajv');
+
+const expect = require('chai').expect;
+const normalizeAjvErrors = require('./normalizeAjvErrors');
+
+describe('#normalizeAjvErrors', () => {
+  const ajv = new Ajv({ allErrors: true });
+  let schema;
+  let userConfig;
+  let ajvErrors;
+
+  it('should show error messages for root and second level property', () => {
+    userConfig = {
+      foo: 'bar',
+      package: { incclude: ['./folder'] },
+    };
+    schema = {
+      type: 'object',
+      properties: {
+        package: {
+          type: 'object',
+          properties: {
+            include: { type: 'array', items: { type: 'string' } },
+          },
+          additionalProperties: false,
+        },
+      },
+      additionalProperties: false,
+    };
+    const validate = ajv.compile(schema);
+    validate(userConfig);
+    ajvErrors = validate.errors;
+    expect(normalizeAjvErrors(ajvErrors, userConfig).map(err => err.message)).to.deep.equal([
+      "Unrecognized property 'foo' on 'root'",
```

Comment:
In tests I believe we should not confirm on messages (as that might have improved grammar, or have typos fixed), but on `error.dataPath` and `error.keyword` (which should remain fixed)

Reply:
I could add confimation on error and keyword. But would also confirm on messages. Lot's of code is writen to build an error message, so it id good to be tested I believe.

Reply:
It may make sense to confirm on message customizations, but I would separate concerns, and test two below things separately
- Test that given error types for given data paths surfaces
- Test that expected error message customizations are 

I would also test each thing with different `it` (for good tests readability). It's fine to cover few cases in one test run, but that e.g. can be done in `before` and then we can write different `it` for each tested concern

Reply:
Thanks, I'll make this